docs/tutorial/part-two/index.md

```diff
@@ -114,43 +114,41 @@ import "./src/styles/global.css"
 // require('./src/styles/global.css')
 ```
 
-> Note: Both CommonJS (`require`) and ES Module (`import`) syntax work here. If you’re not sure which to choose, `import` is usually a good default. When working with files that are only run in a Node.js environment however (like `gatsby-node.js`), `require` will need to be used.
+> नोट: दोनों CommonJS (`require`) और ES मॉड्यूल (`import`) सिंटैक्स यहाँ काम करते हैं। यदि आप निश्चित नहीं हैं कि किसे चुनना है, तो 'इम्पोर्ट' आमतौर पर एक अच्छा डिफ़ॉल्ट विकल्प है। केवल Node.js एन्वाइरन्मेंट में चलाई जाने वाली फ़ाइलों के साथ काम करने के लिए (जैसे `gatsby-node.js`), `require` का उपयोग करने की आवश्यकता होगी।
 
-3. Start the development server:
+3. डेवेलपमेंट सर्वर शुरू करें:
 
 ```shell
 gatsby develop
 ```
 
-If you take a look at your project in the browser, you should see a lavender background applied to the "hello world" starter:
+यदि आप ब्राउज़र में अपनी प्रॉजेक्ट पर एक नज़र डालते हैं, तो आपको "हैलो वर्ल्ड" स्टार्टर पर लागू लैवेंडर बॅकग्राउंड दिखनी चाहिए:
 
 ![Lavender Hello World!](global-css.png)
 
-> Tip: This part of the tutorial has focused on the quickest and most straightforward way to get started styling a Gatsby site — importing standard CSS files directly, using `gatsby-browser.js`. In most cases, the best way to add global styles is with a shared layout component. [Check out the docs](/docs/global-css/) for more on that approach.
+> नोट: ट्यूटोरियल के इस भाग मे सबसे तेज़ और सबसे सरल तरीके से Gatsby साइट('Gatsby-browser.js` का उपयोग करते हुए, सीधे स्टॅंडर्ड सीएसएस फाइलों को इम्पोर्ट करते हुए) को स्टाइलिंग करने पर ध्यान केंद्रित किया गया है। ज्यादातर मामलों में, ग्लोबल सीएसएस को जोड़ने का सबसे अच्छा तरीका एक साझा लेआउट कॉंपोनेंट है। [उस दृष्टिकोण पर अधिक जानकारी के लिए डॉक्स देखें] (/docs/global-css/)।
 
-## Using component-scoped CSS
+## कॉंपोनेंट-स्कोप वाला सीएसएस का उपयोग करना
 
-So far, we've talked about the more traditional approach of using standard css stylesheets. Now, we'll talk about various methods of modularizing CSS to tackle styling in a component-oriented way.
+अब तक, हमने स्टॅंडर्ड सीएसएस स्टाइलशीट का उपयोग करने के अधिक पारंपरिक दृष्टिकोण के बारे में बात की है। अब, हम कॉंपोनेंट-ओरियेनटेड तरीके से सीएसएस को संशोधित करने के विभिन्न तरीकों के बारे में बात करेंगे।
 
-### CSS Modules
+### सीएसएस मॉड्यूल
 
-Let's explore **CSS Modules**. Quoting from
-[the CSS Module homepage](https://github.com/css-modules/css-modules):
+आइए ** सीएसएस मॉड्यूल ** को एक्सप्लोर करें। [सीएसएस मॉड्यूल होमपेज](https://github.com/css-modules/css-modules):
```

Comment:
`[the CSS Module homepage](https://github.com/css-modules/css-modules):` => `[सीएसएस मॉड्यूल होमपेज](https://github.com/css-modules/css-modules) से:`

how about this?

Reply:
ok